dataactvalidator/config/sqlrules/fabs43_detached_award_financial_assistance_4.sql

```diff
@@ -0,0 +1,38 @@
+-- If PrimaryPlaceOfPerformanceCongressionalDistrict is provided it must be valid in the state indicated by
+-- PrimaryPlaceOfPerformanceCode. The PrimaryPlaceOfPerformanceCongressionalDistrict may be 90 if the state has more
+-- than one congressional district or PrimaryPlaceOfPerformanceCode is 00*****
+-- Districts that were created under the 2000 census or later are considered valid for purposes of this rule.
+WITH detached_award_financial_assistance_fabs43_4_{0} AS
+    (SELECT submission_id,
+        row_number,
+        place_of_performance_code,
+        place_of_performance_congr
+    FROM detached_award_financial_assistance
+    WHERE submission_id = {0})
+SELECT
+    dafa.row_number,
+    dafa.place_of_performance_code,
+    dafa.place_of_performance_congr
+FROM detached_award_financial_assistance_fabs43_4_{0} AS dafa
+WHERE CASE WHEN COALESCE(dafa.place_of_performance_congr, '') <> ''
+            THEN ((dafa.place_of_performance_congr <> '90'
+                    AND dafa.row_number NOT IN (
+                        SELECT DISTINCT sub_dafa.row_number
+                        FROM detached_award_financial_assistance_fabs43_4_{0} AS sub_dafa
+                        JOIN state_congressional AS sc_1
+                            ON UPPER(LEFT(sub_dafa.place_of_performance_code, 2)) = sc_1.state_code
+                            AND sub_dafa.place_of_performance_congr = sc_1.congressional_district_no
+                            AND (sc_1.census_year IS NULL
```

Comment:
We can probably do a coalesce here as well instead of a null check `COALESCE(sc_1.census_year, 2000) >= 2000`. I believe that's faster